templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -2504,6 +2504,7 @@ Resources:
             install -o $user -g $user -m 0755 "$tap_dir/downloads/pivnet" /usr/local/bin/pivnet
             echo "Installing tanzu CLI and staging tanzu-cluster-essentials..."
             su - $user -c "$tap_dir/src/install-tools.sh"
+            sleep 120
```

Comment:
Adding sleep is just a workaround and not the fix.
I think I found the cause of this error.

The error 'No cluster found for name:' might happen when .kubeconfig is not updated with the cluster name.

Use below command before running any kubectl commands
`aws eks update-kubeconfig --name ${MY_CLUSTER_NAME}`


You can easily do this with 
`/tap-setup-scripts/src/tap-main.sh -c prereqs ${MY_CLUSTER_NAME} . `
Do this for all clusters. 

Do this for a single cluster as well.